tasks/skylab/StarAlign.wdl

```diff
@@ -66,29 +66,33 @@ task StarAlignBamSingleEnd {
 
 }
 
-task StarAlignFastqPairedEnd {
+task STARsoloFastq {
   input {
-    File fastq1
-    File fastq2
+    Array[File] r1_fastq
+    Array[File] r2_fastq
     File tar_star_reference
+    File white_list
+    String chemistry
+    String input_id
+    String counting_mode
 
     # runtime values
     String docker = "quay.io/humancellatlas/secondary-analysis-star:v2.7.9a"
     Int machine_mem_mb = ceil((size(tar_star_reference, "Gi")) + 6) * 1100
```

Comment:
this probably need to be changed

Reply:
I still think there is a cleaner way to calculate the memory size here, for instance: 
```suggestion
    Int machine_mem_mb = ceil((size(tar_star_reference, "Mi")) + 6) 
```

Reply:
Or just hardcode a default, since the reference is generally a constant size from run to run